doc/languages-frameworks/beam.section.md

```diff
@@ -14,15 +14,15 @@ nixpkgs follows the [official elixir deprecation schedule](https://hexdocs.pm/el
 
 All BEAM-related expressions are available via the top-level `beam` attribute, which includes:
 
-- `interpreters`: a set of compilers running on the BEAM, including multiple Erlang/OTP versions (`beam.interpreters.erlangR22`, etc), Elixir (`beam.interpreters.elixir`) and LFE (Lisp Flavoured Erlang) (`beam.interpreters.lfe`).
+- `interpreters`: a set of compilers running on the BEAM, including multiple Erlang/OTP versions (`beam.interpreters.erlang_22`, etc), Elixir (`beam.interpreters.elixir`) and LFE (Lisp Flavoured Erlang) (`beam.interpreters.lfe`).
 
 - `packages`: a set of package builders (Mix and rebar3), each compiled with a specific Erlang/OTP version, e.g. `beam.packages.erlang22`.
```

Comment:
`beam.packages.erlang22` clearly doesn't work.

Moreover, the whole document really needs to be updated, since Erlang 22 (so LFE 1.3) is marked as insecure in nixpkgs.

Reply:
You mean: `beam.packages.erlang_22` or else it would be `beam.interpreters.erlangR22`.
I'm not aware of `erlang22`.

Reply:
Yes.